### PR TITLE
r-bh is a build-time-only dependency

### PR DESCRIPTION
**Pull request by xhochy**
…and also a quite heavy one.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -57,7 +56,7 @@ test:
 
 about:
   home: https://symbolixau.github.io/spatialwidget/articles/spatialwidget.html
-  license: GPL-3
+  license: GPL-3.0-or-later
```
Comment (dbast) [inline recipe/meta.yaml]: meta data tells (see commented out at the end of the meta.yaml) that this should be `GPL-3.0`.

Comment (xhochy) [reply recipe/meta.yaml]: This is the SPDX form for the license as mentioned by the linter. GPL3 is not sufficient for SPDX.

Comment (xhochy) [reply recipe/meta.yaml]:
```suggestion
  license: GPL-3.0
```

Comment (xhochy) [reply recipe/meta.yaml]: Adjusted but the linter is not happy

**Comment by dbast** [comment]
Strange ... GPL-3.0 should be a valid short identifier according to https://spdx.org/licenses/GPL-3.0.html ... seems like conda-smithy wants GPL-3.0-only: https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/licenses.txt#L168

File: recipe/meta.yaml [hunk 2]
```diff
@@ -57,7 +56,7 @@ test:
 
 about:
   home: https://symbolixau.github.io/spatialwidget/articles/spatialwidget.html
-  license: GPL-3
+  license: GPL-3.0
```
Comment (dbast) [inline recipe/meta.yaml]:
```suggestion
  license: GPL-3.0-only
```

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Pull request merged**